commit afbca92c40b839855c92120c6963ad0ec2f36047 (refs/changes/52/4952/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-10-12T13:36:48-07:00 (12 months ago)
    
    OS-7306 /usr/vm tests should include test name in output
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/src/vm/runtest b/src/vm/runtest
index b87ce942..086f568c 100755
--- a/src/vm/runtest
+++ b/src/vm/runtest
@@ -38,6 +38,8 @@ if [[ -z "$1" ]]; then
     exit 1
 fi
 
+echo "# Running $1"
+
 # Before starting tests, ensure we have the source(s) we want for test VM images
 
 have_sources="imgadm sources | sort"
